scripts/importer.py

```diff
@@ -27,51 +27,171 @@
 
 
 import argparse
+import sys
+
+browser_default_input_format = {
+    'chromium': 'netscape',
+    'ie': 'netscape',
+    'firefox': 'netscape',
+    'seamonkey': 'netscape',
+    'palemoon': 'netscape'
+}
 
 
 def main():
     args = get_args()
-    if args.browser in ['chromium', 'firefox', 'ie']:
-        import_netscape_bookmarks(args.bookmarks, args.bookmark_format)
+    bookmark_types = []
+    output_format = None
+    input_format = args.input_format
+    if args.import_search:
```

Comment:
I'm getting a crash when trying to run this here. Where is `import_search` defined?

```
Traceback (most recent call last):
  File "./importer.py", line 200, in <module>
    main()
  File "./importer.py", line 46, in main
    if args.import_search:
AttributeError: 'Namespace' object has no attribute 'import_search'
```

I think it should be defined by argparse, which argument does it correspond to?

I think it should be `search_output`, right? I'm not familiar with argparse though so I could be wrong

Reply:
Yes, fixed.

I also added documentation to make clear that browsers don't necessarily export every type of bookmark.